### PR TITLE
CAMEL-11044-create list with a single element instead of creating lis…

### PR DESCRIPTION
**Pull request by onders86**
create list with a single element instead of creating list with asList

### CODE REVIEW

File: camel-core/src/main/java/org/apache/camel/util/EndpointHelper.java [hunk 1]
```diff
@@ -363,8 +356,10 @@ public static boolean isReferenceParameter(String parameter) {
                 // The bean is a list
                 return (List) bean;
             } else {
-                // The bean is a list element
-                return Arrays.asList(elementType.cast(bean));
+            	// The bean is a list element
```
Comment (zregvart) [inline camel-core/src/main/java/org/apache/camel/util/EndpointHelper.java]: There seems to be a tab character here, so the Checkstyle check fails with:

    camel-core/src/main/java/org/apache/camel/util/EndpointHelper.java:359:13: File contains tab characters (this is the first instance). [FileTabCharacter]

Can you fix that? You can check with:

    mvn -Psourcecheck verify

Comment (onders86) [inline camel-core/src/main/java/org/apache/camel/util/EndpointHelper.java]: i am not sure why this is happening. I guess something is wrong with my checkstyle plugin on eclipse. cs fix will be coming soon. sorry about that.

**Comment by zregvart** [comment]
Thanks for the fix @onders86, sorry to bug you with the finer details, could you squash those two commits and force push them? Thank you 👍

**Comment by zregvart** [comment]
I think you have accidentally squashed one of Nicolas commits as well 😀 , I've picked only your changes and merged them into `master`. Thank you for your contribution you can close this pull request 👍

**Comment by onders86** [comment]
yes i was trying to remove it to fix the commit. as you already picked the right ones, i can close now. thanks. closing.

**Pull request closed without merging**